chirp/drivers/leixen.py

```diff
@@ -961,14 +961,25 @@ class JetstreamJT270MRadio(LeixenVV898Radio):
     _file_ident = b"JET"
     _model_ident = b'LX-\x89\x85\x53'
 
+class VV898SDualBank(chirp_common.Alias):
+
+    '''Newer VV898S 1.06+ firmware that features dual memory banks'''
+    VENDOR = "Leixen"
+    MODEL = "VV-898S (1.06+ firmware)"
```

Comment:
We really try not to put stuff like this in `MODEL`. If we *must* do this, then it should be in `VARIANT`.

Is there not any way we can detect which model is the right one when talking to the radio? Perhaps reading a chunk of memory to extract a version string or some indication that it's the newer model? It's really awful user experience for people to have to pick "the right one" from the model box and we really try to avoid that whenever possible.